src/daemon/mod.rs

```diff
@@ -246,11 +246,7 @@ pub(super) async fn start(
         genesis_header.clone(),
     )?);
 
-    // Network Upgrade manifests are stored in the blockstore but may not be
-    // garbage collected. Until this is fixed, the GC has to be disabled.
-    // Tracking issue: https://github.com/ChainSafe/forest/issues/4926
-    // if !opts.no_gc {
-    if false {
```

Comment:
Would love to see GC being manually triggered in at least one of the CI tests.  Could you create a tracking issue if it cannot land in this PR?

Reply:
there is an issue for manual gc.